test/integration/test.js

```diff
@@ -590,6 +590,52 @@ describe('integration', function () {
             );
         });
 
+        // doesn't work in PhantomJS
+        if (!/PhantomJS/.test(window.navigator.userAgent)) {
+            it('should bail out if accessing the `type` and `target` properties of an event throw an exception', function (done) {
+                // see: https://github.com/getsentry/raven-js/issues/768
+                var iframe = this.iframe;
+
+                iframeExecute(iframe, done,
+                    function () {
+                        setTimeout(done);
+
+                        // some browsers trigger onpopstate for load / reset breadcrumb state
+                        Raven._breadcrumbs = [];
+
+                        // click <input/>
+                        var evt = {};//document.createEvent('MouseEvent');
+                        MouseEvent.prototype.initMouseEvent.call(evt,
+                            "click",
+                            true /* bubble */,
+                            true /* cancelable */,
+                            window,
+                            null,
+                            0, 0, 0, 0, /* coordinates */
+                            false, false, false, false, /* modifier keys */
+                            0 /*left*/,
+                            null
+                        );
+
+                        function kaboom() { throw new Error('lol'); };
+                        Object.defineProperty(evt, 'type', { get: kaboom });
```

Comment:
Unfortunately in Phantom (and Phantom 2) just calling `Object.defineProperty` on this `MouseEvent` instance triggers an error. I've tried `__defineGetter__` as well – no luck.